### PR TITLE
Switch to SP2

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -522,7 +522,7 @@ for arch in $product_arches; do
         if ! is_ses; then
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: Not visible here, but on line 518, you probably need to change SLES12-SP1 to SLES12-SP2 (unless SP2 still has the bug where it pretends to be SP1...)

File: scripts/install-chef-suse.sh [hunk 2]
```diff
@@ -501,7 +501,7 @@ fi
 
 for arch in $product_arches; do
     # Checks for SLE12 SP1 medias
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: s/SP1/SP2/

File: scripts/install-chef-suse.sh (hunk 1 above)
Comment (rsalevsky) [inline scripts/install-chef-suse.sh]: Fixed

**Comment by jsuchome** [comment]
+1

**Comment by jsuchome** [comment]
I assume this cannot turn green without https://github.com/SUSE-Cloud/automation/pull/995 ...

File: scripts/install-chef-suse.sh [hunk 3]
```diff
@@ -542,10 +542,10 @@ fi
 
 
 for arch in $supported_arches; do
-    check_or_create_ptf_repository 12.1 $arch PTF
+    check_or_create_ptf_repository 12.2 $arch PTF
```
Comment (jsuchome) [inline scripts/install-chef-suse.sh]: I think we still need PTF for 12.1 as well

**Comment by jsuchome** [comment]
See https://github.com/crowbar/crowbar/pull/2228

**Comment by vuntz** [comment]
Was cherry-picked in https://github.com/crowbar/crowbar/pull/2228

**Pull request closed without merging**